vcx/wrappers/python3/vcx/api/proof.py

```diff
@@ -24,7 +24,7 @@ def proof_state(self, x):
         self._proof_state = x
 
     @staticmethod
-    async def create(source_id: str, name: str, requested_attrs: list, revocation_interval: dict):
```

Comment:
What about others vcx wrapper? Do we need updates there?

Reply:
Good catch, I'm not familiar with any of the other wrappers (Java, ios, node) but they would need a similar change.

I don't have a local environment that I can test changes to any of these wrappers.